### ISSUE TITLE
Static Screen Summary: wording and structure changes 

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#231]
The latest information in the screen summary ([dev 1.0.0-dev.71/](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.71/phet/ratio-and-proportion_a11y_view.html)) is scaffolding learners much better to immediately start moving the hands (individual sliders or both hands). Learners are interacting with the hands much more quickly than in previous interviews.

That said, there still room for improvement in wording and chunking, so the information reads better with screen reader software. 

In this issue, I will 
- [x] first post the current content from the latest version, 
- [x] and then post common changes for both screen summaries, 
- [x] then post individual changes for each screen.

**Here's the current content:**

## Current Structure & Content of Static part of the Discover Screen
P1: The interactive Discover Screen changes as you play with it. It has a Play Area and a Control Area.

P2: In the Play Area you have two movable hands and a set of challenge ratios to explore. Hands can be moved up and down individually or simultaneously to explore multiple places where the chosen challenge is true. There are options to add plain or numbered tick marks shown as evenly-spaced horizontal lines that divide up the vertical space of the Play Area. The Control Area has a button to reset the sim.

[[This is where the dynamic part of the screen summary goes]]

P: Move hands individually or at the same time to maintain the challenge ratio.

P: If needed, check out keyboard shortcuts under Sim Resources.

## Current Structure & Content of Static part of the Create Screen
P1: The interactive Create Screen changes as you play with it. It has a Play Area and a Control Area.

P2: On this screen you can set your own challenge ratio. The Play Area has two movable hands that can be moved individually or simultaneously, or as a single interaction with the ratio locked to explore the challenges you create. There are options to add plain or numbered tick marks shown as evenly-spaced horizontal lines that divide up the vertical space of the Play Area. In addition, you can increase the range of tick marks for a finer exploration of your challenges. The Control Area has a button to reset the sim.

[[This is where the dynamic part of the screen summary goes]]

P: Explore current challenge ratio or create your own.

P: If needed, check out keyboard shortcuts under Sim Resources.

**Comment by terracoda** [comment]
## Common changes for both screens fro RaP:
@zepumph , I want to drop the word "interactive" in the first sentence. I want this to be for all multi-screen sims. I am not sure if the first sentence is part of a different repo. The words "interactive screens" will be on the Home Screen see https://github.com/phetsims/joist/issues/665 . Here on the individual screen page of a multi-screen sim, I would like the general pattern for the first sentence to be:

- P1: The {{Name of Screen}} Screen changes as you play with it. It has a Play Area and a Control Area.

Then after that, there is the sim-specific descriptions of the Play Area and Control Area (static and dynamic) that make up the next parts of the screen summary.

@zepumph, also for the sim-specific part of the screen summary, it is good to check things up into small paragraphs. I am pretty sure this has to be done sim by sim and screen by screen, so the re-chunking of content in the P2's will be part of the individual screen changes.

In summary, there is just one common change. - remove "interactive" in the first sentence.

**Comment by terracoda** [comment]
@zepumph, the individual changes are cover chunking and some minor wording changes

## Changes to P2 of the Discover Screen
P2 will change to:

P: The Play Area is a vertical space containing two hands that can be moved up and down. You can move the Left Hand and the Right Hand individually or activate Both Hands to move them both hands at the same time. Your goal is to move the hands up and down to find multiple places where the chosen challenge is true. 

P: There options to add plain or numbered tick marks. The tick marks are evenly-spaced horizontal lines that divide up the vertical play space. When ready, you can choose a different challenge to master.  

P: The Control Area has a button to reset the sim.

**Summary of changes:**
1. reworded everything
2. chunked new content into 3 paragraphs.

**Comment by terracoda** [comment]
## Changes t P2 of Create Screen
P2 will change to:

P: On this screen you can set your own challenge ratio. The Play Area has two movable hands that can be moved individually or simultaneously, or as a single interaction with the ratio locked to explore the challenges you create. There are options to add plain or numbered tick marks shown as evenly-spaced horizontal lines that divide up the vertical space of the Play Area. In addition, you can increase the range of tick marks for a finer exploration of your challenges. The Control Area has a button to reset the sim.

P: On this screen you can set your own challenge ratio. The Play Area is a vertical space containing two hands that can be moved up and down. You can move the hands individually or at the same time using Both Hands. In addition there is a checkbox that provides an option to lock the ratio. When locked that hands move as  single unit. 

P: There are options to add plain or numbered tick marks shown as evenly-spaced horizontal lines that divide up the vertical space of the Play Area. In addition, you can increase the range of tick marks to explore larger numbers. 

P: The Control Area has a button to reset the sim.

[[This is where the dynamic part of the screen summary goes]]

P: Explore current challenge ratio or look for My Challenge to create your own.

**Comment by terracoda** [comment]
@zepumph, I want to run changes to content by @Matthew-Moore240 and @emily-phet.

**Comment by emily-phet** [comment]
I added a suggested change to the bottom of https://github.com/phetsims/ratio-and-proportion/issues/231#issuecomment-724171448. If these suggestions are good, they would need to be propagated into the Create screen description.

**Comment by terracoda** [comment]
@terracoda to re-post all changes.

**Comment by terracoda** [comment]
@zepumph, final changes to P2 of Discover Screen

## Final copy for P2 of the Discover Screen

P: The Play Area is a tall space containing a Left Hand and a Right Hand that can be moved up and down. You can move the Left Hand and the Right Hand individually, or activate Both Hands to move both hands at the same time. As you move the hands up and down, you will get closer to or farther from the Challenge Ratio.

P: There options to add plain or numbered tick marks. The tick marks are evenly-spaced horizontal lines that divide up the vertical space. When ready, you can choose a different challenge to master.  

P: The Control Area has a button to reset the sim.

[[This is where the dynamic part of the screen summary goes]]

P: Move hands to find the Challenge Ratio in different places.

P: If needed, check out keyboard shortcuts under Sim Resources.


For reference, the current help text for the sliders and both hands interactions:
- Sliders: 
    - Move hands up and down to find the challenge ratio in different locations.
- Both Hands: 
    - Use W and S keys to move left hand, and Up and Down Arrow keys to move right hand. Move both hands at the same time to maintain the challenge ratio.

In light, of changes to the screen summary, we may need to tweak the help text on the interactions slightly. I don't think they contradict each other in any way, but may need refinement.

I'll open a new related issue for those changes.

**Comment by emily-phet** [comment]
This looks great to me - I made a minor edit to comment above fixing typo (no content changes).

**Comment by zepumph** [comment]
Discover screen implemented above. I think @terracoda still needs to give a final update on the create screen.

**Comment by terracoda** [comment]
## Final copy for Changes to P2 of Create Screen

P: Here, you can create your own Challenge Ratio. The Play Area is a tall space containing a Left Hand and a Right Hand that can be moved up and down. You can move the Left Hand and the Right Hand individually, or activate Both Hands to move both hands at the same time. As you move the hands up and down, you will get closer to or farther from the Challenge Ratio, unless you lock the ratio. 

P: There are options to add plain or numbered tick marks. The tick marks are evenly-spaced horizontal lines that divide up the vertical space. In addition, you can increase the range of tick marks to explore larger numbers. 

P: The Control Area has a button to reset the sim.

[[This is where the dynamic part of the screen summary goes]]

P: Explore current challenge, or open My Challenge to create your own. 

@emily-phet or @Matthew-Moore240, please review for typos :-)

**Comment by terracoda** [comment]
Assigning to you @zepumph because I am pretty sure there are no typos :-)

**Comment by zepumph** [comment]
Implemented.

**Comment by terracoda** [comment]
@zepumph, the word "interactive" in the first sentence needs to be removed. That first sentence might be in common code in a different repo, see https://github.com/phetsims/ratio-and-proportion/issues/231#issuecomment-724142653.

All other changes are perfect! Thanks!
It's probably safe to close this issue, but I am assigning back to you just in case.

**Comment by zepumph** [comment]
Yes this is in common code. Please confirm that you are alright with this change in every screen summary at PhET. Feel free to close.

**Comment by terracoda** [comment]
@zepumph, the idea was that the removal of "interactive" on the screen page would be for mullti-screen sims, only.

I don't want to remove "interactive" from single screen sims like JT, BASE, Fiction, etc which have this sentence:
Single Screen Sim starts like this:
- {{Gravity Force Lab: Basics}} is an interactive sim. It changes as you play with it. It has a Play Area and a Control Area.

Screen of a multi-screen sim starts like this:
- The {{Discover}} Screen changes as you play with it. It has a Play Area and a Control Area. 

On the Home screen says:
-  "Welcome to {{Ratio and Proportion}}. It has {{2}} interactive screens that you can explore. Choose a screen to start exploring."

We can follow-up this conversation in the common code issue, can you point me the appropriate issue?

**Comment by terracoda** [comment]
I think the issue is phetsims/joist#665

I'll copy the previous comment to that issue

**Comment by zepumph** [comment]
From slack discussion, @terracoda and I want to add back in "interactive". Anything else here?

**Comment by zepumph** [comment]
From more discussion with @terracoda, we want to remove this just for the multi-screen sentence, which is what was done above, but I confused the situation with incorrect info. Now the "interactive" is out for multi-screen sims (this doesn't include Rap with `?screens=1`). @terracoda anything else here?

**Comment by terracoda** [comment]
Looks good. Thanks for making the change again :-) Closing.
